### DISCUSSION TITLE
Choosing between KeyChanger.uc.js and KeyChanger_fx70.uc.js

### DISCUSSION THREAD

**Discussion by Crunchbits** [benzBrake/FirefoxCustomize#34] [Q&A] [answered]
Is one version more compatible/optimized for the latest Firefox version(127) or what separates them apart feature-wise?

**Comment by benzBrake** [comment, marked as answer]
There's no extractly difference between this two scripts currently, but I have no time keeping update this two scripts in the future, please use KeyChanger_fx70.uc.js.
